src/core/agent/services/connectionService.ts

```diff
@@ -386,6 +424,16 @@ class ConnectionService extends AgentService {
       };
     });
   }
+
+  private async getLinkedIPEXMessageByConnectionId(
+    connectionId: string
+  ): Promise<IpexMessageDetails[]> {
+    const messages =
+      await this.ipexMessageStorage.getIpexMessageMetadataByConnectionId(
+        connectionId
+      );
+    return messages;
+  }
```

Comment:
We can just do `const linkedIpexMessages = await this.ipexMessageStorage.getIpexMessageMetadataByConnectionId(connectionId);`

There's no need for another function when its one line